src/index.js

```diff
@@ -97,13 +100,17 @@ FlowWebpackPlugin.prototype.apply = function(compiler) {
         compilation.errors.push('Flow validation' + details)
     })
 
+    if(plugin.options.bundle) {
+        compiler.plugin('done', runCallback(false))
+    }else{
     /*
      * callbacks chosen because it is required
      * * to be done before or at time of 'compilation callback' - to avoid expensive compilation when type error present
      * * hook needs to be asynchronous
```

Comment:
@happylynx  I fixed the issues. But by this comment you mean we shouldn't use 'done' in this case?